### PR TITLE
Show hand controllers in away.js

### PR DESCRIPTION
**Pull request by hyperlogic**
* only works for openvr vive controllers
* unifies display scripts for tutorial and away.js.

### CODE REVIEW

**Comment by hyperlogic** [comment]
QA Test:
  * while wearing a vive hmd.  Enter away mode by pressing ESC
  * the vive controllers should be visible, making it easier to grab them.
  * the vive controllers should not be visible when NOT in away mode.

**Comment by hifi-gustavo** [comment]
Build console output for commit: c431c69d60848ed2372a17f274c532c0e76f00ef available at: https://gist.github.com/fecda5b1dcbb763bb9ac7a329fb862d6<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8880-c431c69.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8880-c431c69.dmg

File: tools/atp-extract.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#
```
Comment (ZappoMan) [inline tools/atp-extract.py]: this should be a different PR

**Comment by RyanDowne** [comment]
This worked as promised. Two things that I am not sure are a result of this PR or if they are specific to the machine I was using..
1) When in the tutorial, the controllers yellow triggers are no longer animated. That's regression
2) When equipping the lighter, the alignment is off after being fixed. That's regression.

**Comment by hifi-gustavo** [comment]
Build console output for commit: ae301602e71e6d74e5173013504a233f22af3285 available at: https://gist.github.com/fecda5b1dcbb763bb9ac7a329fb862d6<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8880-ae30160.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8880-ae30160.dmg

**Comment by hyperlogic** [comment]
@ryanDowne before you evaluate how this works with the tutorial, @huffman will need to update the tutorial assets with this version of tutorial.js.

**Comment by hyperlogic** [comment]
@RyanDowne I should clarify, this change will work fine with the exiting tutorial.  Except, that the assets will potentially loaded twice from separate places if you go into away mode while inside the tutorial.  This PR includes a change to tutorial content that will share the same assets (bundled with the installer) and not bundled with the tutorial content.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 84d7eef4ba701ab9befb729787966ddd59208354 available at: https://gist.github.com/fecda5b1dcbb763bb9ac7a329fb862d6<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8880-84d7eef.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8880-84d7eef.dmg

File: interface/src/scripting/HMDScriptingInterface.cpp [hunk 2]
```diff
@@ -57,6 +57,20 @@ bool HMDScriptingInterface::isHandControllerAvailable() {
     return PluginUtils::isHandControllerAvailable();
 }
 
+void HMDScriptingInterface::requestShowHandControllers() {
```
Comment (ZappoMan) [inline interface/src/scripting/HMDScriptingInterface.cpp]: ok - I guess we accept it for now, but I think you're point about needing some better cross script syncing would be a better way to do this... since basically the C++ doesn't really do anything other than send a signal.

Comment (ZappoMan) [inline interface/src/scripting/HMDScriptingInterface.cpp]: I wonder if a `Script.sendGlobalSignal()` and `Script.connectGlobalSignal()` would be better? thoughts?
